Sources/MapboxNavigation/LaneView.swift

```diff
@@ -36,6 +40,14 @@ extension LaneIndication {
             primaryIndication = .sharpLeft
         } else if indications.contains(.sharpRight) && maneuverDirection ?? .sharpRight == .sharpRight {
             primaryIndication = .sharpRight
+        } else if indications.contains(.left) && !indications.contains(.sharpLeft) && !indications.contains(.uTurn) && maneuverDirection ?? .sharpLeft == .sharpLeft {
+            primaryIndication = .left
+        } else if indications.contains(.right) && !indications.contains(.sharpRight) && !indications.contains(.uTurn) && maneuverDirection ?? .sharpRight == .sharpRight {
+            primaryIndication = .right
```

Comment:
This is tricky. A distinction between sharp left, left, and left doesn’t matter much in general – a sharp left maneuver should bias the ranking towards whatever is on the left. However, a user is likely to distinguish between a left and a U-turn to the left.